### DISCUSSION TITLE
Please add potential improvements here

### DISCUSSION THREAD

**Discussion by Swedemon** [Swedemon/MzFightReporter#18] [General]
Feel free to add suggestions/improvements here.  Try to be specific and check back here for feedback/questions.

**Comment by ReisenBagels** [comment]
Would it be possible to move the Enemy summary to be above the Enemy Top Damage By Skill section? This way, segregate the Allies from the Enemy data for an easy-to-read solution?

Or perhaps customizable layout options on the UI of how the data is output, with examples to output to the console?

**Comment by Swedemon** [comment]
A workable solution is to provide a configurable list you can adjust. Is a
bit of work but could be worth.
